tests/integration-tests/tests/health_check/test_scheduled_events/test_scheduled_events/pcluster.config.ini

```diff
@@ -0,0 +1,19 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = {{ initial_queue_size }}
+maintain_initial_size = true
```

Comment:
do we want to have the ASG guarantee the min size or make sure we do that regardless of ASG?

Reply:
Since the main point of the integration test is to test end-to-end behavior of locking and terminating a node because of scheduled event, I don't think `maintain_initial_size` setting alters the nature of the test. However, having `maintain_initial_size` is nice for executing consecutive tests on the same cluster and I think makes the test more consistent.